README.md

```diff
@@ -16,7 +16,7 @@ Contains the common transformer (T-Engine) code, plus a few actual implementatio
 
 The project can be built by running the Maven command:
 ```bash
-mvn clean install -Plocal
+mvn clean install -DskipTests -Plocal
```

Comment:
Requires ActiveMQ to be running and configured at the moment, so you might like to work out what needs to change in the README.md, before closing ATS-484.

Reply:
Yes, we should update the README steps (as needed) so that we do not skip tests when building locally. It might be as a separate PR.

Reply:
I've updated the line with the proper profiles.